prow/gerrit/adapter/adapter.go

```diff
@@ -238,6 +245,7 @@ func (c *Controller) Sync() {
 		}()
 
 		changes := c.gc.QueryChangesForInstance(instance, syncTime, c.config().Gerrit.RateLimit)
+		timeCompletedGettingChanges := time.Now()
```

Comment:
Nit: rename to `timeQueryingForChanges` or maybe even `timeQueryChangesForInstance` to match the function call name.

Reply:
Done.